doc/2/api/controllers/document/exists/index.md

```diff
@@ -0,0 +1,59 @@
+---
+code: true
+type: page
+title: exists
+---
+
+# exists
+
+Checks if a document exists.
```

Comment:
Maybe there is a missing `Since 2.0.0` badge here

Reply:
We didn't add `Since 2.0.0` badge for other methods, shall we ?

Reply:
@alexandrebouthinon is right, we should add "since" badges as often as possible.

Reply:
Fixed